app/src/main/java/org/simple/clinic/activity/placeholder/PlaceholderScreen.kt

```diff
@@ -5,15 +5,22 @@ import android.util.AttributeSet
 import android.widget.RelativeLayout
 import io.reactivex.android.schedulers.AndroidSchedulers
 import io.reactivex.disposables.Disposable
-import kotlinx.android.synthetic.main.screen_placeholder.view.*
 import org.simple.clinic.await.Await
 import org.simple.clinic.await.Checkpoint
+import org.simple.clinic.databinding.ScreenPlaceholderBinding
 import java.util.concurrent.TimeUnit.SECONDS
 
 class PlaceholderScreen(context: Context, attrs: AttributeSet) : RelativeLayout(context, attrs) {
   private val delayToShowMessage = SECONDS.toMillis(3).toInt()
   private val await = Await(listOf(Checkpoint.unit(delayToShowMessage)))
   private var awaitDisposable: Disposable? = null
+  private val binding: ScreenPlaceholderBinding? = null
```

Comment:
This will cause a crash, the binding is never being assigned at all. We need to load and clear the binding in accordance with the view lifecycle.

Please see an existing screen like `EditMedicinesScreen` for a reference implementation.